docs/GettingStartedGuide.md

```diff
@@ -33,7 +33,7 @@ The following two authentication approaches are available within your applicatio
 
 2. Using your **app access key id** and **secret access key**, which you can find on platform's [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
 
-After you set up the credentials, you can then use them to access HERE OLP SDK libraries.
+After you set up the credentials, you can use them to access HERE OLP SDK libraries.
```

Comment:
Change to:
"After you set up the credentials, you can use them to access the HERE OLP SDK libraries."